pkgs/applications/misc/keepassx/community.nix

```diff
@@ -90,7 +91,7 @@ stdenv.mkDerivation rec {
     runHook postCheck
   '';
 
-  nativeBuildInputs = [ asciidoctor cmake wrapQtAppsHook qttools pkg-config ];
+  nativeBuildInputs = [ asciidoctor cmake wrapGAppsHook wrapQtAppsHook qttools pkg-config ];
```

Comment:
This double wraps the binary if I am not mistaken. Please add the wrapper arguments to wrapperArgs.